runtime/Ansible.py

```diff
@@ -452,6 +469,11 @@ def unpackage(data):
                             printStackTrace=False))
                     break
                 unpackaged_data = unpackage(recv_data) # pylint: disable=unused-variable
+                if unpackaged_data.header == notification_pb2.Notification.TIMESTAMP_DOWN:
+                    timestamps = list(unpackaged_data.timestamps)
+                    timestamps.append(time.monotonic())
+                    state_queue.put([HIBIKE_COMMANDS.TIMESTAMP_DOWN, timestamps])
+                    continue
                 state_queue.put([SM_COMMANDS.STUDENT_UPLOAD, []])
```

Comment:
Since we are handling messages that are not just messages informing runtime that student code was uploaded, we should add a check for this line and only send to state manager if we actually receive a student upload command. We should check if the header is equivalent to the STUDENT_SENT enum defined in notification.proto.